README.md

```diff
@@ -411,6 +485,21 @@ The permission corresponds to an "Edition Permission" in Yudu Publisher.
 </permission>
 ```
 
+##### POST
```

Comment:
How does this work? Don't you need to provide a PDF etc?

Reply:
@rmanthorpe documentUrl should be a url to the location of the pdf to be published.

Reply:
This `documentUrl` thing is a little confusing actually. It could do with some explanation here. If `documentUrl` part of the `EditionState` when you *GET* and edition? It seems like ephemeral information, not part of the representation of the state of an edition and therefore doesn't make sense in a REST API.

Reply:
The `storedFile` style resource as in #20 would make more sense than a `documentUrl`.